### ISSUE TITLE
Wrong parameter types for some routines

### ISSUE THREAD

**Issue by J-F-Liu** [stainless-steel/lapacke#2]
`?ggev` functions should be:
```rust
#[inline]
pub unsafe fn sggev(
    layout: Layout,
    jobvl: u8,
    jobvr: u8,
    n: i32,
    a: &mut [f32],
    lda: i32,
    b: &mut [f32],
    ldb: i32,
    alphar: &mut [f32],
    alphai: &mut [f32],
    beta: &mut [f32],
    vl: &mut [f32],
    ldvl: i32,
    vr: &mut [f32],
    ldvr: i32,
) -> i32 {
    ffi::LAPACKE_sggev(
        layout.into(),
        jobvl as c_char,
        jobvr as c_char,
        n,
        a.as_mut_ptr(),
        lda,
        b.as_mut_ptr(),
        ldb,
        alphar.as_mut_ptr(),
        alphai.as_mut_ptr(),
        beta.as_mut_ptr(),
        vl.as_mut_ptr(),
        ldvl,
        vr.as_mut_ptr(),
        ldvr,
    )
}

#[inline]
pub unsafe fn dggev(
    layout: Layout,
    jobvl: u8,
    jobvr: u8,
    n: i32,
    a: &mut [f64],
    lda: i32,
    b: &mut [f64],
    ldb: i32,
    alphar: &mut [f64],
    alphai: &mut [f64],
    beta: &mut [f64],
    vl: &mut [f64],
    ldvl: i32,
    vr: &mut [f64],
    ldvr: i32,
) -> i32 {
    ffi::LAPACKE_dggev(
        layout.into(),
        jobvl as c_char,
        jobvr as c_char,
        n,
        a.as_mut_ptr(),
        lda,
        b.as_mut_ptr(),
        ldb,
        alphar.as_mut_ptr(),
        alphai.as_mut_ptr(),
        beta.as_mut_ptr(),
        vl.as_mut_ptr(),
        ldvl,
        vr.as_mut_ptr(),
        ldvr,
    )
}
```

**Comment by IvanUkhov** [comment]
Thank you for noticing. Please feel free to change [`generate.py`](https://github.com/blas-lapack-rs/lapacke/blob/master/bin/generate.py) and open a pull request.

**Comment by selvavm** [comment]
@IvanUkhov - I am also in need of this. Can you tell me how to modify generate.py? (sorry my knowledge is limited in here)

**Comment by IvanUkhov** [comment]
I have added a section on development to the readme. In order to fix the types, one has to open `bin/generate.py` and turn `alphar`, `alphai`, `beta`, `vl`, and `vr` to non-scalars in the `is_scalar` function.

**Comment by IvanUkhov** [comment]
Fixed in #5.
